material/guias/bash.md

```diff
@@ -0,0 +1,42 @@
+# Bash
+
+1. Contar la cantidad de palabras de un archivo. Si no existe, devolver 0.
+
+1. Dadas las notas de del tp1 y tp2 en dos variables, quiero saber si aprobé o no (aprobar implica que el promedio de las notas sea mayor o igual a `7`).
+
+1. Dada una cantidad indeterminada de notas, valores que mi script recibe por parámetro, deseo saber si aprobé o no (aprobar implica que el promedio de las notas sea mayor o igual a `7`).
```

Comment:
Me gusta, pero lo  movería para abajo para que vaya de temas fáciles a difíciles ya que esto usa un for.

Reply:
Te parece más díficil que los de abajo?